Sample code for the Complex Numbers tutorial

bzaczynski opened the pull request:
**Where to put new files:**

- New files should go into a top-level subfolder, named after the article slug. For example: `my-awesome-article`

**How to merge your changes:** 

1. [Make sure the CI code style tests all pass (+ run the automatic code formatter if necessary).](https://github.com/realpython/materials/blob/master/README.md)
2. Find an RP Team member on Slack and ask them to review & approve your PR.
3. Once the PR has one positive ("approved") review, GitHub lets you merge the PR.
4. 🎉

somacdivad reviewed (changes requested):
These are really cool notebooks @bzaczynski!

I found one small issue in the `bermuda.ipynb` file where you used the word **negate** to mean "take the negative of", which we removed from the article. We should update that here, too.

Also, I would normally argue against using `global` like you have in the `fourier.ypynb` file, but I understand why you did it in this context. I wonder if we should add a note in a Markdown cell explaining that although this is typically considered poor practice, there is a practical reason you're doing it here. What do you think?

bzaczynski commented:
@somacdivad Thanks for catching this. I added a Jupyter Notebook's alert box to warn about the global keyword and changed the README file as there's no need to install Pillow anymore.

somacdivad reviewed (approved):
Thanks for the updates @bzaczynski! Everything looks good!
